libcxx/test/std/numerics/simd/simd.class/simd.ctor/range.pass.cpp

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <simd>
+
+// REQUIRES: std-at-least-c++26
+
+#include <array>
+#include <cassert>
+#include <numeric>
+#include <simd>
+#include <type_traits>
+
+#include "type_algorithms.h"
+#include "../../utils.h"
+
+namespace dp = std::datapar;
+
+template <class T>
```

Comment:
Can we add a `assert.range.pass.cpp` test in `libcxx/test/libcxx` to check that we have an assertion for the  alignment precondition?